sorc/build_wsr88d.sh

```diff
@@ -21,4 +21,6 @@ make
 cd $pwd/wsr88d_level2.fd/; 
 make
 
-cp wsr88d_level2 ../../exec 
+mkdir -p -m 775 ../../exec
```

Comment:
why `-p 775`?

Reply:
NCO made this change. I think they want to set permission for directory in operation.

Reply:
so they want `group-writable` permissions to the `exec` directory? 
Seems odd to me.

Reply:
NCO made similar changes in 
JRADAR_LEVEL2:mkdir -p -m 775 ${COMOUT_SRCtank}
JRADAR_LEVEL2:    chmod 775 ./poescript$proc
JRADAR_LEVEL2:chmod 775 poescript
JRADAR_REFLECTIVITY_MOSAIC:    chmod 775 ./poescript$proc
JRADAR_REFLECTIVITY_MOSAIC:chmod 775 poescript
I guess this allows that more than one developer at NCO is able to check radarl2 output with their individual account.